### DISCUSSION TITLE
在HuggingFace页面点击Duplicate this Space按钮后，提示Internal Error - We're working hard to fix that as soon as possible!

### DISCUSSION THREAD

**Discussion by Qwerty4401** [GaiZhenbiao/ChuanhuChatGPT#480] [Q&A]
陆续试了快1个月，一直这样提示，大佬们有什么解决办法吗？

**Comment by GaiZhenbiao** [comment]
试试factory reboot？在space的settings页面里。

**Reply by Qwerty4401** [reply]
还没复制成功，在填写链接那里就报错了。。

**Reply by Qwerty4401** [reply]
还有create new space那里也是，点Create Space按钮也直接报这个错。

**Reply by GaiZhenbiao** [reply]
没有地方需要填写链接吧

**Reply by Qwerty4401** [reply]
![微信截图_20230423140535](https://user-images.githubusercontent.com/102785432/233822954-5c57f6c2-1486-4dae-ac8f-bf972824f83a.png)
就是这里会报这个错。
